app/src/main/kotlin/com/softteco/template/data/base/ProfileDataBase.kt

```diff
@@ -0,0 +1,52 @@
+package com.softteco.template.data.base
```

Comment:
I think the place for this file and the `dao` and `model` packages is in the `device` package. This package is meant to be the package for the basic (generic) entities of the data layer